keps/sig-scheduling/5055-dra-device-taints-and-tolerations/README.md

```diff
@@ -258,30 +306,85 @@ Device and node taints are applied independently. A node taint applies to all
 pods on a node, whereas a device taint affects claim allocation and only those
 pods using the claim.
 
+### kubectl describe DeviceTaintRule
+
+An extension of `kubectl describe` for DeviceTaintRule uses the same code as
+the eviction controller to calculate which devices are matched and which pods
+would need to get evicted. The input for that code is the single
+DeviceTaintRule which gets described and the complete set of ResourceSlices,
+ResourceClaims and Pods.
+
+Calculating the outcome on the client side was chosen because the alternative,
+always updating the status of DeviceTaintRules on the server side, would cause
+permanent churn regardless whether someone wants the information or not.
+
+The exact output will be decided during the implementation phase.
+It can change at any time because the `kubectl describe` output does not fall
+under any Kubernetes API stability guarantees.
+
 ### API
 
-The ResourceSlice content gets extended:
+The ResourceSlice content gets extended. A natural place for a taint that
```

Comment:
Since slices can now be “devices” or “taints”, can we call out:

- How the scheduler/DRA controllers index taint-slices vs device-slices.
- Cache invalidation and event fan-out when taint-slices update, to avoid O(N×M) reprocessing.

Reply:
Short answer: the ResourceSlice tracker keeps track of that state and tries to minimize the changes that the consumers (scheduler and eviction controller) see.

The long answer is a bit more involved and I'd like to punt on that now while still in alpha. It needs to be added for beta when discussing scalability.